### PR TITLE
Added Photo Upload to Post Creation Module.

### PR DESCRIPTION
**Pull request by xReddskie**
OVERVIEW

User can upload photo.
Migration for photos.

SNIPPET
image

NOTES
Not yet fully integrated with UI

### CODE REVIEW

File: src/app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -6,25 +6,39 @@
...
+                $filename = time() . '.' . $image->getClientOriginalExtension();
+                $path = $image->storeAs('public/post_images', $filename); // Capture the path
+
+                Photo::create([
+                    'user_id' => $userId,
+                    'post_id' => $post->id,
+                    'img_file' => $path // Use the stored path
+                ]);
+            }
+        }
```
Comment (hipe01mark) [inline src/app/Http/Controllers/PostController.php]: Create a seperate service layer or library class for the Image processes

File: src/app/Models/Photo.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Photo extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['user_id', 'post_id', 'img_file'];
+
+    public function user()
```
Comment (mikco-hipe) [inline src/app/Models/Photo.php]: add return type

File: src/app/Models/Photo.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        return $this->belongsTo(User::class);
+    }
+
+    public function post()
```
Comment (mikco-hipe) [inline src/app/Models/Photo.php]: add return type

File: src/app/Http/Controllers/PostController.php [hunk 4]
```diff
@@ -6,25 +6,26 @@
...
      * Create user posts
      */
     public function create(PostRequest $request): RedirectResponse
     {
-        $userId = auth()->id(); 
-        $posts = $this->postService->create($request, $userId);
-        
+        $userId = auth()->id();
+        $post = $this->postService->create($request, $userId);
+        $this->postService->addPhoto($request, $userId, $post);
```
Comment (hipe01mark) [inline src/app/Http/Controllers/PostController.php]: This is multiple images. it should be addPhotos. It must be plural if the method is doing multiple requests.

**Review by mikco-hipe — APPROVED** [review]
LGTM

**Pull request merged**